src/dipdup/dipdup.py

```diff
@@ -74,11 +74,7 @@ async def run(
                     spawn_datasources_event.set()
 
             if start_scheduler_event and not start_scheduler_event.is_set():
-                # NOTE: Do not check with every_index_is, indexes become REALTIME after first message from WS is received
-                for index in self._indexes.values():
-                    if index.state.level != index.datasource.head.level:
-                        break
-                else:
+                if self._every_index_is(IndexStatus.REALTIME):
```

Comment:
What about indexes that are oneshot?
When I've "finished" indexing I end up having both REALTIME and ONE_SHOT indexes, this would make it impossible for jobs to fire if every INDEX must be REALTIME, maybe "not SYNCHING" would be better.
But I would honestly like this option to be optional, ie, I would like to run jobs while "SYNCHING"